tests/test_video_downloader.py

```
@@ -0,0 +1,20 @@
+import pytest
+from src.utils.downloader import VideoDownloader
+
+
+@pytest.fixture
+def downloader() -> VideoDownloader:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)